### ISSUE TITLE
JupyterLab Heuristic Evaluation

### ISSUE THREAD

**Issue by grey280** [jupyter/design#63]
I’m part of a group from UC Irvine, working on designing a new file browser experience. As part of this work, we’ve been doing a lot of research on the current state of JupyterLab. 
While it’s mostly focused on the file browsing/managing experience, the [heuristic evaluation](https://www.nngroup.com/articles/how-to-conduct-a-heuristic-evaluation/) we’ve completed could be of some use to the community, so we’re sharing it here.

At the moment, [I’m linking to this as stored in my personal .github.io repo](https://github.com/grey280/grey280.github.io/blob/master/JupyterLab-HeuristicEvaluation.pdf), for lack of a better place, but are hoping @tgeorgeux can help us get this stored somewhere more appropriate/permanent for use by the Jupyter community.

Here’s hoping this is helpful, and thank you for your time.

Oh, and for reference, the group consists of @jabumeri, @JDLEarley, @emilythefan, @Nastraughn, and myself.

**Comment by blink1073** [comment]
Thank you very much for this work @grey280, @jabumeri, @JDLEarley, @emilythefan, and @Nastraughn!   These type of insights are invaluable to improve our product.

Can you please give the output of `jupyter labextension list` to give the version of JupyterLab and any installed extensions used in the analysis?

**Comment by blink1073** [comment]
And I use "our" here to include you as well, this is a community project after all :wink:.

**Comment by grey280** [comment]
Been learning to use `conda` while we're doing this; on my machine, I was mostly using:
```
JupyterLab v2.1.0
Known labextensions:
   app dir: /Users/greypatterson/opt/miniconda3/envs/jupyter-fs/share/jupyter/lab
        jupyterlab_filetree v0.2.2  enabled  OK
        multicontentsmanager v0.0.1  enabled  OK
```
but did occasionally switch to the 'base' state, which was just:
```
JupyterLab v2.1.0
No installed extensions
```

@jabumeri, @JDLEarley, I believe you two contributed most of the screenshots, what do your setups look like?

**Comment by JDLEarley** [comment]
Thank you @blink1073! We are happy to be a part of this project and look forward to assisting the community. I was running JuptyerLab 2.1.0 with no extensions.

**Comment by jabumeri** [comment]
Yes, thank you @blink1073! Your excitement about our work thus far is encouraging and to echo Justin, we're excited to be part of this project. 
Here is my output when running `jupyter labextension list`:

```
JupyterLab v2.1.0
Known labextensions:
   app dir: /Users/jabumeri/opt/miniconda3/share/jupyter/lab
        jupyterlab_filetree v0.2.2  enabled  OK
        multicontentsmanager v0.0.1  enabled  OK
``` 

I have primarily been running JupyterLab with the enable extensions and have not switched to the base version like Grey.

**Comment by blink1073** [comment]
Thanks @grey280, @JDLEarley, @jabumeri.  The filetree extension was what I was most interested in, since we may decide to replace the stock filebrowser with that one and this feedback can help guide that decision.
